journal/original/15-chapter-12.adoc

```diff
@@ -1385,6 +1418,9 @@ And a great care being upon me, lest any young or ignorant people,
 that might sometimes come amongst us, should be drawn into that snare,
 I was moved to give forth the following epistle as a warning unto all such:--
 
+[.embedded-content-document.letter]
```

Comment:
Let's go for `epistle`, since it is for Friends generally, and he uses the word epistle above, as well.